docs/.vuepress/components/ModuleGroups.vue

```diff
@@ -37,7 +37,7 @@
         icon="wifi"
         link="./titanium/network.html"
       >
-        Root namespace for network related code, such as an <router-link to="./titanium/network/httpclient.html">HTTP client</router-link>.
```

Comment:
Could we switch these to `a` links maybe?

Reply:
I'll give it a try! One sec

Reply:
yes and no. Links are shown correctly but the anchors only work if you open the link in a new tab. Just clicking on it will link the the page. So the camera/gallery link below will just link to meda anyway.
I'll add the link to the HTTP Client page back in

Reply:
sorry, have to roll back! It will always open the card link!

Reply:
No problem!